examples/chat/module/rust/tests/integration_test.rs

```diff
@@ -154,8 +144,13 @@ impl<'a> Chatter<'a> {
             user_handle,
             base64::encode(&room_key_pair.pkcs8_public_key())
         );
-        let (channel, interceptor) = oak_tests::public_channel_and_interceptor().await;
-        // TODO(#1357): Use key pair to authenticate this client and label requests.
+
+        // Use key pair to label requests and authenticate this client.
+        let (channel, label_interceptor) =
+            oak_tests::authenticated_channel_and_interceptor(&room_key_pair.pkcs8_public_key())
+                .await;
```

Comment:
AFAICT this does not create an authenticated client, it does not even use the private key at all, right? The next interceptor is the one that does authentication, while this does only labelling.

BTW we should probably have a short-hand interceptor that does these two things together, I think it is a very common use case.